packages/wdio-crossbrowsertesting-service/crossbrowsertesting-service.d.ts

```diff
@@ -1,5 +1,12 @@
+/// <reference types="webdriver"/>
+/// <reference types="webdriverio"/>
+
 declare module WebdriverIO {
     interface ServiceOption extends CrossBrowserTestingConfig {}
+
+    interface Browser {
+        jobData: any;
```

Comment:
What does this define?

Reply:
This defines the `jobData` type in the Browser interface which is being used at `service.ts` `updateJob` method

Reply:
no one or no other code part is using `browser.jobData` so we can remove the line and the assignment. It is also wrongly implemented in the testingbot service which this got copy&paste from.

Reply:
Ok, I have removed the jobData assignment from `service.ts` and from the types as well